u8timeseries/models/fft.py

```diff
@@ -0,0 +1,201 @@
+from .autoregressive_model import AutoRegressiveModel
+from ..timeseries import TimeSeries
+from ..custom_logging import raise_if_not, time_log, get_logger
+from ..models.statistics import check_seasonality
+import numpy as np
+import pandas as pd
+from statsmodels.tsa.stattools import acf
+from typing import List, Optional
+
+logger = get_logger(__name__)
+
+def check_approximate_seasonality(series: TimeSeries, seasonality_period: int, 
```

Comment:
I guess this method (and the following ones below) are not meant to be called from outside this file, so I would prepend them with `_`.

Reply:
done